### ISSUE TITLE
Index.UpdateSettingsAsync uses incorrect method (POST instead of PATCH)

### ISSUE THREAD

**Issue by MegaThorx** [meilisearch/meilisearch-dotnet#285]
**Description**
Index.UpdateSettingsAsync uses incorrect method (POST instead of PATCH).

https://docs.meilisearch.com/reference/api/settings.html#update-settings

**Expected behavior**
Due the incorrect method the meilisearch server responds with status 405

**Current behavior**
Settings should be saved

**Screenshots or Logs**

Code:
```csharp
var index = await client.GetIndexAsync(IndexName);

await index.UpdateSettingsAsync(new Settings()
{
    FilterableAttributes = new [] { "Type" },
    SearchableAttributes = new [] { "Title", "Description" }
});
```

Meilisearch log:
```
[2022-07-11T18:47:48Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /indexes/resources HTTP/1.1" 200 123 "-" "Meilisearch .NET (v0.10.2)" 0.000369
[2022-07-11T18:47:48Z INFO  actix_web::middleware::logger] 127.0.0.1 "POST /indexes/resources/settings HTTP/1.1" 405 0 "-" "Meilisearch .NET (v0.10.2)" 0.000043
```

**Environment (please complete the following information):**
- OS: Windows 11
- Meilisearch version: 0.28.0
- meilisearch-dotnet version: 0.10.2

**Comment by alallema** [comment]
Hi @MegaThorx,
Indeed the v0.28.0 of Meilisearch is released yesterday but the SDK is not yet updated with this. The v0.10.2 of this SDK is only compatible with the v0.27, I let you check the [changelog](https://github.com/meilisearch/meilisearch-dotnet/releases) about it.
I will try to update it as soon as possible. Sorry for the inconvenience.

**Comment by MegaThorx** [comment]
Yeah I realized it shortly after opening the issue. I downloaded the older version of Meilisearch and it works fine ;)

**Comment by alallema** [comment]
This issue was fixed in the v0.11.0
